src/resources/BaseInterfaces.ts

```diff
@@ -43,3 +43,8 @@ export interface GranularResource {
     apiKeysThatCanEdit?: IdAndDisplayNameModel[];
     groupsThatCanEdit?: IdAndDisplayNameModel[];
 }
+
+export interface Paginated {
+    page?: number;
+    perPage?: number;
+}
```

Comment:
Do you think it would be relevant to have a "default" JSDoc for each parameters of an interface included in that file?

Reply:
Indeed, actually, I noticed that later on while doing #454 (which starts from this PR), but was too lazy to backport the change to this branch.

Given that both PR are about to be merge in this 🚂 , I'll leave it as is.